Networking/Networking/Remote/WordPressThemeRemote.swift

```diff
@@ -31,6 +36,41 @@ public final class WordPressThemeRemote: Remote, WordPressThemeRemoteProtocol {
         let request = DotcomRequest(wordpressApiVersion: .mark1_1, method: .get, path: path)
         return try await enqueue(request, mapper: WordPressThemeMapper())
     }
+
+    public func installTheme(_ theme: WordPressTheme,
```

Comment:
Nit: since we need only the theme ID for the installation, should we send just the theme ID to the method?

Reply:
I agree. Done in https://github.com/woocommerce/woocommerce-ios/pull/11430/commits/c070358bc01510ab92f25d10adad96fc32d788b9